### PR TITLE
Adds password methods to myAccount API

### PR DESCRIPTION
**Pull request by jaredperreault-okta**

### CODE REVIEW

File: lib/myaccount/README.md [hunk 1]
```diff
@@ -16,14 +16,20 @@ okta.myAccount.profile.manage
 
 **email:**
 ```
-okta.myaccount.email.read
-okta.myaccount.email.manage
+okta.myAccount.email.read
```
Comment (shuowu-okta) [inline lib/myaccount/README.md:19]: due to backend change or previous typo?

File: docs/myaccount/README.md [hunk 2]
```diff
@@ -1,152 +1 @@
 @okta/okta-auth-js/myaccount / [Exports](modules.md)
-
-# MyAccount API
```
Comment (shuowu-okta) [inline docs/myaccount/README.md:3]: where this part goes?

**Review by shuowu-okta — COMMENTED** [review]
Can you also add some simple integration tests to make sure apis are hit properly? see code in test/integration/spec

File: lib/myaccount/README.md (hunk 1 above)
Comment (jaredperreault-okta) [reply lib/myaccount/README.md:19]: I believe this was a typo: https://github.com/okta/okta-auth-js/blob/master/samples/generated/react-embedded-auth-with-sdk/src/config.js#L16

File: docs/myaccount/README.md (hunk 2 above)
Comment (jaredperreault-okta) [reply docs/myaccount/README.md:3]: I'm actually not sure, all I did was run `yarn docs:myaccount` after making my changes. I didn't notice this deleted section

**Comment by jaredperreault-okta** [comment]
There is a follow-up ticket for updating the samples and writing e2e tests (I didn't realize they were separate tickets until I had already done the sample work)

File: test/integration/spec/myaccount/password.ts [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      if (password && password.delete) {
+        try {
+          await password.delete();
+        } catch {
+          // do nothing
+        }
+      }
+    });
+
+    it.only('can manage password with transaction functions', async () => {
```
Comment (jaredperreault-okta) [inline test/integration/spec/myaccount/password.ts:47]: this should be a lint error

**Review by shuowu-okta — APPROVED** [review]

**Pull request merged**